src/main/kotlin/racingcar/game/domain/Participant.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar.game.domain
+
+class Participant(numOfCar: Int) {
+    val cars = arrayListOf<Car>()
```

Comment:
[rangeTo](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)와 map을 활용하여 init을 없애보세요!

```suggestion
    (1..numOfCar).map { Car() }
```

Reply:
[일급컬렉션](https://jojoldu.tistory.com/412)에 대한 의미와 해당 객체를 일급컬렉션으로 만들어보세요!